packages/driver/src/cypress/runner.js

```diff
@@ -889,6 +889,16 @@ const create = function (specWindow, mocha, Cypress, cy) {
     return _testsById[id]
   }
 
+  function hasTestAlreadyRun (test) {
+    if (Cypress._RESUMED_AT_TEST) {
+      if (+test.id.slice(1) < +Cypress._RESUMED_AT_TEST.slice(1)) {
```

Comment:
i would add an additional `getId` function since thats what the `slice(1)` is doing, but its completely masked by this implementation. it looks arbitrary otherwise.